R/update-news.R

```diff
@@ -305,9 +310,20 @@ harvest_pr_data <- function(message) {
     )
   }
 
+  pr_sender <- pr_info$head$repo$owner$login
+
+  external_ctb <- NA_character_
+  if (!is.null(pr_sender)) {
+    repo_owner <- sub("/.*", "", github_slug(get_remote_name()))
+    if (pr_sender != repo_owner) {
+      external_ctb <- pr_sender
+    }
+  }
+
   tibble::tibble(
     title = pr_info$title %||% NA_character_,
-    pr_number = pr_number
+    pr_number = pr_number,
+    external_ctb = external_ctb
```

Comment:
This is not unheard of, to simplify addition of new lines:

```suggestion
    external_ctb = external_ctb,
```